### PR TITLE
String format 281

### PR DESCRIPTION
**Pull request by sgrieve**
Simple PR to standardise all of the string formatting as described in #281

### CODE REVIEW

File: python/nammu/controller/NammuController.py [hunk 1]
```diff
@@ -249,11 +250,11 @@ def saveFile(self, event=None):
         try:
             self.writeTextFile(self.currentFilename, atfText)
         except:
-            self.logger.error("There was an error trying to save %s.",
-                              self.currentFilename)
+            self.logger.error("There was an error trying to "
+                              "save {}}.".format(self.currentFilename))
```
Comment (raquel-ucl) [inline python/nammu/controller/NammuController.py]: Double closing curly brackets.

File: python/nammu/utils/__init__.py [hunk 2]
```diff
@@ -192,34 +192,31 @@ def update_yaml_config(path_to_jar, yaml_path, path_to_config, verbose=False,
                     tmp = {}  # for the nested dics
                     for sub_key in jar_config[key]:
                         if sub_key in local_config[key]:
-                            logger.debug("%s: %s: %s --> Using local values.",
-                                         key,
-                                         sub_key,
-                                         jar_config[key][sub_key])
+                            logger.debug("{}: {}: {}} --> Using local values"
+                                         ".".format(key, sub_key,
```
Comment (raquel-ucl) [inline python/nammu/utils/__init__.py]: Double closing curly brackets.

**Comment by raquel-ucl** [comment]
I'm slightly concerned that I can't test this properly. I don't think we should merge it for this release.

File: python/nammu/utils/__init__.py (hunk 2 above)
Comment (sgrieve) [reply python/nammu/utils/__init__.py]: Wow, I would have thought these would have been flagged on loading Nammu. I'll fix them for this PR, but agree we shouldn't merge this PR at present.

**Comment by dpshelio** [comment]
retest this please

**Comment by sgrieve** [comment]
I'm closing this PR as nammu development has moved on and this will likely never be merged.

**Pull request closed without merging**